airflow/configuration.py

```diff
@@ -775,6 +775,13 @@ def _create_future_warning(name: str, section: str, current_value: Any, new_valu
     def _env_var_name(self, section: str, key: str) -> str:
         return f"{ENV_VAR_PREFIX}{section.replace('.', '_').upper()}__{key.upper()}"
 
+    def get_section_and_key_for_env(self, env: str):
```

Comment:
I don't get why this is needed. If we need a list (again, we already have a concept of sensitive config values I think) then we should specify the list items as `(section,key)` etc